js/src/components/google-ads-account-card/connect-ads/index.js

```diff
@@ -131,7 +136,9 @@ const ConnectAds = ( props ) => {
 							isSecondary
 							disabled={ ! value }
 							eventName="gla_ads_account_connect_button_click"
-							eventProps={ { id: Number( value ) } }
+							eventProps={ getEventProps( {
+								id: Number( value ),
```

Comment:
The `gla_ads_id` is one of the global event properties on the frontend, and the `id` of this event exists before the global event property. In order to avoid unnecessary inconsistencies in tracking data, the current preference is to leave it unchanged.